tests/integration/test_features.py

```diff
@@ -234,6 +227,23 @@ def test_path_params_mapped_in_api(smoke_test_app, apig_client):
     }
 
 
+def _poll_for_resource_id(apig_client, rest_api_id,
+                          max_attempts=10, delay=5):
```

Comment:
Sleep is hardcoded on the other one of these you recently submitted. Make them consistent? Make a generic poller for use in the tests? Seems like we may be adding more of these.

Reply:
Pushed an update that refactored the common logic.  For now I'd prefer to keep the constants local to each retry call until we see a more generalized pattern for what these values should be.